ucp/__init__.py

```diff
@@ -20,6 +20,14 @@
     logger.debug("Setting env UCX_MEMTYPE_CACHE=n, which is required by UCX")
     os.environ["UCX_MEMTYPE_CACHE"] = "n"
 
+if os.environ.get("UCX_CUDA_IPC_CACHE", "") != "n":
```

Comment:
I think we should make it _overwriteable_, such that we only sets `UCX_CUDA_IPC_CACHE` if it is not already set.
E.g. 
```python
if "UCX_CUDA_IPC_CACHE" not in os.environ:
    ...
``` 
Similarly, let's also make `UCX_MEMTYPE_CACHE`_overwriteable_:
```python
if "UCX_MEMTYPE_CACHE" not in os.environ:
    ...
```
It is very useful to be able to compare the performance impact of the cache options.

Reply:
Sounds good.  I'll take care of this -- Peter is on PTO